dev/tools/gen_keycodes/data/keyboard_key.tmpl

```diff
@@ -76,7 +76,7 @@ abstract class KeyboardKey with Diagnosticable {
 ///       _message = 'Pressed the "Q" key!';
 ///     } else {
 ///       if (kReleaseMode) {
-///         _message = 'Not a Q: Key label is "${event.logicalKey.keyLabel}"';
+///         _message = 'Not a Q: Pressed 0x${event.logicalKey.keyId.toRadixString(16)}';
```

Comment:
Maybe print the character or debugLabel here in addition?  Otherwise it doesn't show which key was pressed, just a hex value.

Reply:
I would if this is a framework code. Since it's only an example, I'm aiming for simplicity.

Reply:
Yeah, but it's exactly in an example that it should be clear that you can print those things. People aren't going to print the hex string, they're going to print the debug string so that know what key was pressed, so that's the best example of the usage.

Reply:
Well, the problem is that the `keyId` is technically not connected with the character it represents. It's entirely coincidental that the key ID mirror the ASCII code they represent in the ASCII printable range. How should we convert `0x01000c018a` into a string that says "launch mail"? In release mode all explanatory information of a key object will and should be stripped.

Reply:
Well, if it's part of the Unicode ID plane, it could at least include that Unicode character as a label.